src/Alerts/atom/Modal.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { ReactElement, useCallback, useRef } from 'react';
+import ReactDom from 'react-dom';
+import styled, { css } from 'styled-components'
+import { resetButtonStyles } from '../../common';
+import Icon from '../../Icons/Icon';
+import { useClickOutside } from '../../hooks/useClickOutside';
+
+
+const Container = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: hsl(0, 0%, 0%, 0.2);
+  font-family: ${({ theme }) => theme.fontFamily.ui};
+`;
+
+const CloseIcon = styled(Icon)`
+  display: flex;
+`;
+
+const CloseButton = styled.button<{ selected?: boolean }>`
+  ${resetButtonStyles};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  right: 0;
+  top: -30px;
+  color: ${({ theme }) => theme.colors.icons['dimmed']};
+  font-size: 14px;
+  font-weight: 500;
+
+  ${CloseIcon} {
+    margin-left: 12px;
+  }
+  ${({ selected = false }) => selected && css`
+    border-bottom: 5px solid hsl(207, 80%, 64%);
+  `}
+  &:focus {
+    outline: none;
+  }
+ 
+  &:hover:enabled {
+    ${({ theme }) => theme && css`
+      opacity: .8;
+      transition: transform ${theme.animation.speed.normal} ${theme.animation.easing.primary.inOut};
+    `}
+  }
+
+  &:active:enabled {
+    opacity: .9;
+  }
+  &:disabled {
+    opacity: 0.1;
+  }
+`;
+
+const LightBox = styled.div<{ padding?: string, width?: string, borderRadius?: string, border?: string }>`
+  position: relative;
+  min-height: 300px;
+  margin: 27px 0 0;
+  box-shadow: 0 10px 15px 0 hsla(205, 42%, 60%, 0.15);
+  background-color: hsl(0, 0%, 100%);
+  z-index: 99;
+  width: ${({ width }) => width ? width : `580px`};
+  padding: ${({ padding }) => padding ? padding : `30px 40px`};
+  border-radius: ${({ borderRadius }) => borderRadius ? borderRadius : `10px`};
+  border: ${({border}) => border ? border : `solid 1px hsl(0, 14%, 90%)`};
+`;
+
+// export interface IModalStyleProps {
+
+// }
```

Comment:
Can delete these to tidy up a little! Feel free to point out those kind of things on my PRs anytime by the way. I always miss the tidy up moment 😆